SPEC/NAME.md

```diff
@@ -2,6 +2,9 @@
 
 * [name.publish](#namepublish)
 * [name.resolve](#nameresolve)
+* [name.pubsub.cancel](#namepubsubcancel)
+* [name.pubsub.state](#namepubsubstate)
+* [name.pubsub.subs](#namepubsubsubs)
```

Comment:
Could we re-order these so they're alphabetical with the others? I know this isn't the case for other spec docs but it's something I'd like to start doing. It just makes it easier to find docs for a particular method if you already know it's name.

Reply:
done!